knowledge_base/HABITAT.md

```diff
@@ -1,4 +1,61 @@
 # Working with [`habitat-lab`](https://github.com/facebookresearch/habitat-lab)
 
-We are currently working on integrating habitat-lab environments into torchrl.
-Stay tuned for more info on this.
+## Setting up your environment for habitat and torchrl
+
+### Installing habitat-lab from pip
+
+Instructions can be found on [habitat github repo](https://github.com/facebookresearch/habitat-lab).
+
+  1. **Preparing conda env**
+
+     Assuming you have [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) installed, let's prepare a conda env:
+     ```bash
+     export MY_TEST_ENV=habitat_test
+     conda deactivate
+     conda env remove -n $MY_TEST_ENV -y
+     conda create -n $MY_TEST_ENV python=3.7 cmake=3.14.0 -y
+     conda activate $MY_TEST_ENV
```

Comment:
These were the commands I used because I kept recycling the same environment over and over for testing. I think for instructions, it will be simpler to do this : 
```suggestion
     conda create -n habitat python=3.7 cmake=3.14.0 
     conda activate habitat
```